test/modules/image.spec.ts

```diff
@@ -144,6 +151,16 @@ describe('image', () => {
     });
   });
 
+  describe('personPortrait', () => {
+    it('should return a random avatar url from AI', () => {
+      const imageUrl = faker.image.personPortrait();
+
+      expect(imageUrl).toBeTypeOf('string');
+      expect(imageUrl).toMatch(/^https:\/\/cdn\.jsdelivr\.net\/.*\.jpg$/);
```

Comment:
Since the size is not randomized, please include the correct default value in the expected regex.

```suggestion
      expect(imageUrl).toMatch(/^https:\/\/cdn\.jsdelivr\.net\/.*\/512\/.*\.jpg$/);
```

Reply:
Maybe also include the other fixed parts like repo name.

Reply:
And maybe use `(female|male)` and `/d{1,2}` for index.